src/components/navigation/course-footer.tsx

```diff
@@ -29,7 +31,22 @@ export function CourseFooter({
   const { strings } = useInstanceData()
 
   return (
-    <nav className="mt-10 mb-8 py-5 bg-brand-50 sm:bg-white">
+    <nav className="mt-10 mb-8 py-5 bg-brand-50 sm:bg-white flex justify-between">
+      {previousHref ? (
+        <Link
+          href={previousHref}
+          path={['courseback']}
+          className="serlo-button serlo-make-interactive-primary mx-side hover:no-underline"
+        >
+          <FontAwesomeIcon
+            icon={faArrowCircleRight}
+            className="fa-flip-horizontal"
```

Comment:
nice hack :D
we use `faArrowCircleLeft` in the breadcrumbs so I think this could be considered premature optimisation ^^
okay either way